server/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -2299,4 +2299,26 @@ public void nested_partition_column_in_doc() {
         execute("select _doc from part order by id");
         assertThat(response.rows()[0][0]).isEqualTo(Map.of("id", 1, "name", "Ford", "obj", Map.of("x", "a")));
     }
+
+    @Test
+    public void test_primary_key_in_pg_index() {
+        execute("create table doc.t1 (id int, primary key (id)) partitioned by (id);");
+        execute("select i.indexrelid, i.indrelid, i.indkey, i.indnkeyatts from pg_index i, pg_class c where c.relname = 't1' and c.oid = i.indrelid;");
+        assertThat(response).hasRows("-649073482| 728874843| [1]| 1");
```

Comment:
Does it cover FULLTEXT indices like 
`CREATE TABLE t1(a text, b text, INDEX ft USING fulltext(a, b))`?

What PG returns for such table?

Reply:
This is not relevant to this change, as currently we populate the table only with PKs anyway.

Reply:
And `pg_index` is initialized in https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/execution/engine/collect/sources/InformationSchemaIterables.java#L246